### PR TITLE
adding a textArea to see versionInfo into it.

### PR DESCRIPTION
**Pull request by steph37tours**
the textArea is initialized in controller.

I add a textArea to see the versionInfo

- [ ] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Screenshots added (for bigger UI changes)
- [ ] Manually tested changed features in running JabRef
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)
- [ ] If you changed the localization: Did you run `gradle localizationUpdate`?

### CODE REVIEW

File: src/main/java/org/jabref/gui/help/AboutDialogController.java [hunk 1]
```diff
@@ -68,4 +75,9 @@ public void openLicense() {
     public void openDonation() {
         viewModel.openDonation();
     }
+
+    public void setTextAreaVersions(TextArea textAreaVersions) {
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/help/AboutDialogController.java]: This method is not used, right?

File: src/main/java/org/jabref/gui/help/AboutDialogViewModel.java [hunk 2]
```diff
@@ -38,6 +40,7 @@
     private final DialogService dialogService;
     private final ReadOnlyStringWrapper developmentVersion = new ReadOnlyStringWrapper();
     private final ClipBoardManager clipBoardManager;
+    private final TextArea textAreaVersions = new TextArea();
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/help/AboutDialogViewModel.java]: This is also not used right?

File: src/main/java/org/jabref/gui/help/AboutDialogController.java (hunk 1 above)
Comment (steph37tours) [reply src/main/java/org/jabref/gui/help/AboutDialogController.java]: Yes I remove it.

File: src/main/java/org/jabref/gui/help/AboutDialogViewModel.java (hunk 2 above)
Comment (steph37tours) [reply src/main/java/org/jabref/gui/help/AboutDialogViewModel.java]: Yes @Siedlerchr , I remove it too.

File: src/main/java/org/jabref/gui/help/AboutDialogController.java [hunk 3]
```diff
@@ -1,8 +1,11 @@
 package org.jabref.gui.help;
 
+
```
Comment (lenhard) [inline src/main/java/org/jabref/gui/help/AboutDialogController.java]: That is one empty line too much, which is also one of the reasons why the travis build fails

File: src/main/java/org/jabref/gui/help/AboutDialogViewModel.java [hunk 4]
```diff
@@ -8,7 +8,6 @@
 import javafx.beans.property.ReadOnlyBooleanWrapper;
 import javafx.beans.property.ReadOnlyStringProperty;
 import javafx.beans.property.ReadOnlyStringWrapper;
-
```
Comment (lenhard) [inline src/main/java/org/jabref/gui/help/AboutDialogViewModel.java]: There should be an empty line here, which is also one of the reasons why the travis build fails.

File: src/main/java/org/jabref/gui/help/AboutDialogViewModel.java [hunk 5]
```diff
@@ -29,6 +28,7 @@
     private static final String LICENSE_URL = "https://github.com/JabRef/jabref/blob/master/LICENSE.md";
     private final String changelogUrl;
     private final String versionInfo;
+    private final ReadOnlyStringWrapper environmentInfo = new ReadOnlyStringWrapper();
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/gui/help/AboutDialogViewModel.java]: The `environmentInfo` is not needed (or set), right?

Comment (steph37tours) [reply src/main/java/org/jabref/gui/help/AboutDialogViewModel.java]: I did not change that. environmentInfo is use line 37 in AboutDialog.fxml

**Comment by lynyus** [comment]
@steph37tours, there are some minor code style issues. I recommend you to use https://github.com/JabRef/jabref/blob/master/config/eclipseJabRef.xml as you eclipse code style config file. This should format most things correctly.

Also you run `gradlew check` on a console before pushing a commit. In your case a checkstyle check has failed:
```
:checkstyleMain[ant:checkstyle] [ERROR] /home/travis/build/JabRef/jabref/src/main/java/org/jabref/gui/help/AboutDialogViewModel.java:11: 'org.jabref.gui.AbstractViewModel' should be separated from previous imports. [ImportOrder]
 FAILED
```

I know this is a lot to learn, feel free to ask for assistance in the slack channel.

**Pull request merged**